lib/reply.js

```diff
@@ -197,6 +198,13 @@ Reply.prototype.headers = function (headers) {
 }
 
 Reply.prototype.code = function (code) {
+  if (!code ||
```

Comment:
Should not be a little bit faster the opposite check?

`if(!(code && (code < 601 || statusCodes[code]))`

Reply:
I agree, also, there is no need to use the type check, as JavaScript will cast automatically any number to string when doing a lookup in the object.

Reply:
Anyhow, you can directly write:
```js
if (statusCodes[code] === undefined) {
  throw new FST_ERR_BAD_STATUS_CODE()
}
```
Because `statusCodes[null] === undefined` and `statusCodes[undefined] === undefined` ;)